core/src/main/java/com/justai/aimybox/api/DialogApi.kt

```diff
@@ -116,7 +117,7 @@ abstract class DialogApi<TRequest : Request, TResponse : Response> :
             val nextAction = Aimybox.NextAction.byQuestion(response.question)
 
             val speeches = response.replies
-                .filter { it is TextReply || it is AudioReply }
+                .filter { (it is TextReply)  || it is AudioReply }
```

Comment:
Why you did not wrap `it is AudioReply`  the same way?

Reply:
Ops. It is an artifact.